2021-02-01-infinispan-cli-image.adoc

```diff
@@ -0,0 +1,49 @@
+---
+layout: blog
+title: Infinispan Native CLI
+permalink: /blog/:year/:month/:day/infinispan-12-final
+date: '2021-02-01T12:00:00.000-00:00'
+author: remerson
+tags: [ "docker", "cli", "containers", "quarkus", "native"]
+---
+
+Starting with Infinispan 12, we now provide a natively compiled version of the Infinispan CLI.
+
+This can be consumed as a native binary or via a container image.
+
+== Usage
+. Start a Infinispan server instance so that the CLI has an endpoint to connect to:
+----
+docker run --net=host -it -p 11222:11222 -e USER="user" -e PASS="pass" quay.io/infinispan/server:12.0
+----
+
+==== Container
+[start=2]
+. Launch the CLI image and connect to the server image:
+
+----
+docker run --net=host -it --rm quay.io/remerson/cli -c http://127.0.0.1:11222
+----
+[start=3]
+. Enter the "user" and "pass" credentials, for the username and password respectively.
+
+TIP: The container's endpoint is the CLI binary, therefore it's possible to pass CLI args straight to the image,e.g. `infinispan/cli:12.0 --version`
```

Comment:
```suggestion
TIP: The container's endpoint is the CLI binary, therefore it's possible to pass CLI args straight to the image,e.g. `quay.io/infinispan/cli:12.0 --version`
```

Reply:
maybe use the full command? `docker run --net=host --rm quay.io/infinispan/cli:12.0 --version`